static/js/search_suggestion.js

```diff
@@ -109,10 +110,22 @@ function get_stream_suggestions(last, operators) {
     const hilite = typeahead_helper.highlight_with_escaping_and_regex;
 
     const objs = streams.map((stream) => {
-        const prefix = "stream";
         const highlighted_stream = hilite(regex, stream);
-        const verb = last.negated ? "exclude " : "";
-        const description = verb + prefix + " " + highlighted_stream;
+        let description = "";
+        // Here escape value is set to false so that stream names with some letters as highlighted
+        // are rendered correctly otherwise suggestions like Exclude stream verona are rendered as
+        // Exclude stream <strong>v</strong><strong>e</strong>...
+        if (last.negated) {
+            description = i18n.t("exclude stream __stream__", {
+                stream: highlighted_stream,
+                interpolation: {escapeValue: false},
```

Comment:
This can be.
```js
 description = i18n.t("stream __- stream__", {
    stream: highlighted_stream,
});
```
Also in other places where you don't want to escape the variables.
https://zulip.readthedocs.io/en/latest/translating/internationalization.html#frontend-translations

Reply:
Made these changes.